### DISCUSSION TITLE
Overwhelmed by notifications is another notification I can't turn off

### DISCUSSION THREAD

**Discussion by Tyriar** [community/community#5267] [Projects and Issues] [answered]
Every few months I'm asked this:

![image](https://user-images.githubusercontent.com/2193314/130609196-44b2718f-822c-46ce-aeb1-08f9b0642b05.png)

This is essentially yet another notification I have to deal with by dismissing manually and it's kind of frustrating. I look after my watched repos already - pressing x should permanently dismiss it for the account imo.

**Comment by omerbensaadon** [comment, marked as answer]
We unshipped this :)
